backend/src/server_process_helper.py

```diff
@@ -27,6 +28,9 @@ def _port_in_use(port: int):
         return s.connect_ex(("127.0.0.1", port)) == 0
 
 
+SANIC_LOG_REGEX = r"\[.*\] \[\d*\] \[(\w*)\] (.*)"
```

Comment:
You should anchor the pattern so it doesn't in the middle of the string.

> `\[.*\]`

`.` also matches `[` and `]`, so this may end up matching waaaay too much.

```suggestion
SANIC_LOG_REGEX = r"^\[[^\[\]]\] \[\d*\] \[(\w*)\] (.*)"
```

Reply:
Nice edit lol I was about to mention that the original didn't work right